src/main/python/lbrynetservice.py

```diff
@@ -124,34 +123,23 @@ def start():
 
     lbrynet_log = conf.settings.get_log_filename()
     log_support.configure_logging(lbrynet_log, True, [])
-    log.debug('Final Settings: %s', conf.settings.get_current_settings_dict())
+
+    # TODO: specify components, initialise auth
+    conf.settings.update({
+        'components_to_skip': ['peer_protocol_server', 'reflector'],
```

Comment:
Would be better to use the constants for these strings in case we change them and that way it'll fail fast (`NameError`) instead of more complicated errors coming from component manager.

https://github.com/lbryio/lbry/blob/d4a7dbf80576b3d354198496d966f36ca1d0b78a/lbrynet/daemon/Components.py#L35-L48